armadillo1/TROUBLESHOOTING.md

```diff
@@ -0,0 +1,29 @@
+# Armadillo stack troubleshooting
+
+### "become" needs to work
+When you login to a VM you are hopefully yourself as in a useraccount that is recognisable as your account. After you logged in you need to be able to perform `sudo su` without entering a password. Get that in place and you will be able to run the playbook.
+
+### Ansible installation: debian.yml not found
+When you are deploying the armadillo stack with ansible, it could happen that you get this error saying that a file is not found. This is an known bug. If you have access to the system, please go to `/usr/share/armadillo` and type the following command for RHEL/CentOS systems: `podman-compose up -d`, for the other systems type: `docker-compose up -d`.
+
+### Use of own certificate file(s)
+If you want to use HTTPS for an secure connection you will need to get SSL certificate files. You could also add a SSL wildcard certificate file(not every company/institute allows that). You will need to do the following things:
+1. Get the certificate files(domain.cer/domain.key)
+2. When you have received the certificate files you will need to copy them on the server
+3. Make a backup of the files located in `/etc/nginx/default.d`(RHEL/CentOS) or `/etc/nginx/sites-available`(Debian/Ubuntu) with an extension other then conf
+4. We have given example configuration files for SSL termination, copy the the files given in `/etc/nginx/examples/` to `/etc/nginx/sites-available`(Debian/Ubuntu) or `/etc/nginx/default.d`(RHEL/CentOS)
+5. Edited the armadille_ssl.conf file copied in step 4 and add the location of the certificate file to `ssl_certificate` and add the location of the key file to `ssl_certificate_key`
+6. Repeat the process for all the *_ssl.conf files
+7. After editing the files, check if everything is looking file with the command: `nginx -t`
+8. If there are no problems displayed, restart nginx with `systemctl restart nginx`
+9. Test if the configuration is working with RStudio
+
+### I can not connect to the server
+1. If you go to the commandline of the local machine and type `nslookup <armadillo domain>`. You should get an A record back with an IP address. If not, you need to ask the IT department to let the domains(Armadillo, MinIO, MinIO console and authentication application) go to the server.
```

Comment:
instead of: If you go to the commandline

Login to the server and type nslookup ..